ambry-store/src/main/java/com.github.ambry.store/BlobStore.java

```diff
@@ -240,6 +245,9 @@ public void put(MessageWriteSet messageSetToWrite) throws StoreException {
             }
             FileSpan fileSpan = new FileSpan(indexEntries.get(0).getValue().getOffset(), endOffsetOfLastMessage);
             index.addToIndex(indexEntries, fileSpan);
+            for (IndexEntry newEntry : indexEntries) {
+              blobStoreStats.handleNewPut(newEntry.getValue());
```

Comment:
For now we do not persist anything, new scan is triggered "immediately", but the actual execution is sequential per disk since there is only one thread per disk for index scanner.